ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/RecoveryTest.java

```diff
@@ -556,6 +560,24 @@ public void shouldRecoverDrop() {
     shouldRecover(commands);
   }
 
+  @Test
+  public void shouldNotDeleteTopicsOnRecovery() {
```

Comment:
Can we add a test (somehow) to prove KSQL doesn't throw a hissy fit on recovery if the command topic contains a legacy `DROP` statement that contains a `DELETE TOPIC`.

i.e. a test to prove this change is backwards compatible.

Reply:
Good catch!

Reply:
Added the test in `KsqlEngineTest` as `shouldIgnoreLegacyDeleteTopicCommand`